### PR TITLE
Psp 1840 refactor frontend layouts, add left hand sidebar for navigation.

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

**Comment by devinleighsmith** [comment]
remove all existing sidebar code as it is not being used and conflicts with the new functionality.

File: frontend/src/_variables.module.scss [hunk 1]
```diff
@@ -2,7 +2,6 @@
 @import './variables';
 
 :export {
-  navbarHeight: $navbar-height;
```
Comment (devinleighsmith) [inline frontend/src/_variables.module.scss:5]: top navbar no longer exists.

File: frontend/src/_variables.scss [hunk 2]
```diff
@@ -1,6 +1,5 @@
 @import './colors.scss';
 
-$header-height: 72px;
-$navbar-height: 45px;
-$mapfilter-height: 60px;
-$footer-height: 46px;
+$header-height: 4.5rem;
```
Comment (devinleighsmith) [inline frontend/src/_variables.scss:3]: continue to try and support scalable ui.

File: frontend/src/components/layout/SideNavBar/NavIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+/**
+ * Component that creates a nav, with an icon, and optional text.
+ * @param {INavIconProps} param0
+ */
+export const NavIcon = ({ icon, text, showText, onClick, roles }: INavIconProps) => {
+  const { hasRole } = useKeycloakWrapper();
+  const displayIcon = _.find(roles, (role: Roles) => hasRole(role));
+
+  return !roles?.length || displayIcon ? (
+    <Nav.Item onClick={onClick} data-testid={`nav-tooltip-${text}`}>
```
Comment (devinleighsmith) [inline frontend/src/components/layout/SideNavBar/NavIcon.tsx:25]: These links don't do anything yet, will update in psp-1843 to actually trigger navigation.

File: frontend/src/features/account/LoginStyled.tsx [hunk 4]
```diff
@@ -6,15 +6,15 @@ import styled from 'styled-components';
...
-    `calc(100vh - ${props.theme.css.footerHeight} - ${props.theme.css.headerHeight})`};
+  position: relative;
   background-image: ${props =>
     props.theme.tenant.login.backgroundImage
       ? `url("${props.theme.tenant.login.backgroundImage}")`
       : ''};
   background-size: cover;
   background-position: center;
   overflow: auto;
+  grid-area: content;
```
Comment (devinleighsmith) [inline frontend/src/features/account/LoginStyled.tsx:17]: the grid handles the vertical sizing, so no explicit height required here.

**Review by asanchezr — APPROVED** [review]

**Pull request merged**